### ISSUE TITLE
Typo in organization name

### ISSUE THREAD

**Issue by strogonoff** [ietf-ribose/relaton-data-misc#5]
https://github.com/ietf-ribose/relaton-data-misc/blob/1f9465d443de097c8be6dac02f1db2b74eddace0/data/reference.IEEE.802.1Qcc-2018.yaml#L29-L40

And the same situation, “publisher” is specified as IETF, although the document is IEEE. If Relaton data says who published _the document itself_, not who published a resource with bibliographic item about that document, then saying an IEEE document is published by IETF is an error.

**Comment by ronaldtse** [comment]
Thanks @strogonoff , there are several problems in this file (and this dataset in general)
1. The publisher is not IETF.
2. There is a typo in the organization name anyway
3. The IEEE documents are available in the relaton-data-ieee data set.
4. This `misc` dataset should be considered static now and no longer mirroring the legacy bibxml copy. We should now edit this dataset manually to correct errors.

**Comment by strogonoff** [comment]
@ronaldtse Alternatively, instead of mapping manually, we can just remove documents from bibxml-misc and adjust [automatic logic for resolving bibxml-misc XML paths](https://dev.bibxml.org/static/docs/_modules/xml2rfc_compat/fetchers.html#misc) so that it does smarter parsing for filenames and has a higher chance of automatically finding IEEE references we may have indexed from other sources.

It should be confirmed by hand though, some of those IEEE XML filenames can be weird and not resolve automatically, requiring a manual map.

**Comment by ronaldtse** [comment]
@strogonoff right, but there is an additional consideration that there are only 94 IEEE entries, so a manual map is easier to do than having to tweak a translation pattern 😉 

So for IEEE documents, we will:
1. Remove the IEEE documents from the `misc` data set
2. Create manual mappings for those documents to the `ieee` data set

OK?

**Comment by ronaldtse** [comment]
@strogonoff can I also remove all the files that are prefixed with `_` now in the `misc` data set?

**Comment by strogonoff** [comment]
> @strogonoff can I also remove all the files that are prefixed with `_` now in the `misc` data set?

I don’t see any such paths here: https://github.com/ietf-ribose/relaton-data-misc/tree/main/data

**Comment by ronaldtse** [comment]
> I don’t see any such paths here: https://github.com/ietf-ribose/relaton-data-misc/tree/main/data

Ah okay.

**Comment by strogonoff** [comment]
I checked with [the XML file](https://github.com/ietf-ribose/bibxml-data-misc/blob/main/data/reference.IEEE.802.1Qcc-2018.xml) that seems to be the source for this, and there is no typo there.

Ping @ronaldtse @andrew2net

**Comment by andrew2net** [comment]
@strogonoff I'll fix it in the next relaton-bib release
